stories/type-scale.scss

```diff
@@ -0,0 +1,101 @@
+/**
+ * Type scale stories stylesheet.
+ *
+ * Site Kit by Google, Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+@import '../assets/sass/config/variables';
+
+.googlesitekit-type-scale {
+	font-family: var(--type-scale-f);
+
+	.googlesitekit-widget__body {
+		display: flex;
+		flex-direction: column;
+		gap: 24px;
+	}
+
+	h2, p {
+		margin: 0;
+	}
+
+	p.lg {
+		font-size: var(--type-scale-fs-lg);
+		line-height: var(--type-scale-lh-lg);
+	}
+	p.md {
+		font-size: var(--type-scale-fs-md);
+		line-height: var(--type-scale-lh-md);
+	}
+	p.sm {
+		font-size: var(--type-scale-fs-sm);
+		line-height: var(--type-scale-lh-sm);
+	}
+
+	&-display {
```

Comment:
Same comment applies as for `tokens.scss` regarding avoiding selector concatenation.